### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by jchollon** [civitai/civitai#585]
### What happened?

I cannot connect.  The website doesn't recognize my instance. 

![image](https://github.com/civitai/civitai/assets/7988067/4d8d1bd2-f983-4403-bec2-4bca34c82d96)

![image](https://github.com/civitai/civitai/assets/7988067/5359f7ce-1a85-49e9-9ad8-9d6244d9420d)

If I refresh the page I can get into the "Get Started" wizard, but when it goes to generate a link key it just spins. 
![image](https://github.com/civitai/civitai/assets/7988067/5cdd68d6-6edb-4866-8b4b-f42d4a43d344)

The dev console shows this

![image](https://github.com/civitai/civitai/assets/7988067/852fb11c-4771-45b0-85ff-b34af0a74da1)

the extension on my server is showing this in the console. 
![image](https://github.com/civitai/civitai/assets/7988067/23e23cd7-d012-481d-a203-86680f8bd67d)

### Steps to reproduce the problem

1. Go to ....
2. Press ....
3. ...

### What should have happened?

I should be able to connect .

### What platforms do you use to access the site?

Windows

### What browsers do you use to access the site?

Google Chrome

### Additional information, context and logs

991075380440a0f2f9b95a84b91a62f90a
Civitai Link: Presence update: SD: 1, Clients: 0
Civitai Link: Disconnected from Civitai Link Server
Civitai Link: Error connecting to Civitai Link Server
Civitai Link: Connected to Civitai Link Server
Civitai Link: Joined room 946ddd1fdbfe8622846752a15c50eccca02b66a950626bb799beb6073c64c0b54280f7cb9e150eb2c54db04b963acc991075380440a0f2f9b95a84b91a62f90a
Civitai Link: Presence update: SD: 1, Clients: 0
Civitai Link: Disconnected from Civitai Link Server
Civitai Link: Connected to Civitai Link Server
Civitai Link: Joined room 946ddd1fdbfe8622846752a15c50eccca02b66a950626bb799beb6073c64c0b54280f7cb9e150eb2c54db04b963acc991075380440a0f2f9b95a84b91a62f90a
Civitai Link: Presence update: SD: 1, Clients: 0
Civitai Link: Disconnected from Civitai Link Server
Civitai Link: Error connecting to Civitai Link Server
Civitai Link: Connected to Civitai Link Server
Civitai Link: Joined room 946ddd1fdbfe8622846752a15c50eccca02b66a950626bb799beb6073c64c0b54280f7cb9e150eb2c54db04b963acc991075380440a0f2f9b95a84b91a62f90a
Civitai Link: Presence update: SD: 1, Clients: 0
Civitai Link: Disconnected from Civitai Link Server
Civitai Link: Connected to Civitai Link Server
Civitai Link: Joined room 946ddd1fdbfe8622846752a15c50eccca02b66a950626bb799beb6073c64c0b54280f7cb9e150eb2c54db04b963acc991075380440a0f2f9b95a84b91a62f90a
Civitai Link: Presence update: SD: 1, Clients: 0
Civitai Link: Disconnected from Civitai Link Server
Civitai Link: Connected to Civitai Link Server
Civitai Link: Joined room 946ddd1fdbfe8622846752a15c50eccca02b66a950626bb799beb6073c64c0b54280f7cb9e150eb2c54db04b963acc991075380440a0f2f9b95a84b91a62f90a
Civitai Link: Presence update: SD: 1, Clients: 0

**Comment by AndrewJames201** [comment]
same problem here

**Comment by jchollon** [comment]
Yeah.  I'm about to pull my support.   I get this is a new feature, but people are paying to support it.  I'm basically paying for broken code and no feedback.
